### ISSUE TITLE
Pretrained Networks using Edge Maps

### ISSUE THREAD

**Issue by sanjeelparekh** [ESanchezLozano/GANnotation#13]
Hi @ESanchezLozano, 

Would it be possible for you to share generator and discriminator networks trained on your datasets using edge maps? I am interested in fine-tuning these on a small dataset that I have. Thanks!

**Comment by ESanchezLozano** [comment]
I will need to dig into the old folders where I hold the experiments I did for this, so this will take a while. You can try to remind me of this in three weeks time? Now it will be hard to find the time to find the trained models.

**Comment by sanjeelparekh** [comment]
It'll be great if it is possible before that as I am working with a deadline 2 weeks later! But of course, I understand that you must be busy and digging into old folders can be tedious. So just to get an idea, if you are not in a position to share these models: would you expect training from scratch with approx. 2400 images (65 unique subjects / ~35 pose+expression images per subject) to give reasonable results?

**Comment by ESanchezLozano** [comment]
Yes I believe that if you play with the augmentation strongly enough you will get something decent. You can in any case try to augment the data with some public datasets or by running a facial landmark detector on an extra set of images.

**Comment by sanjeelparekh** [comment]
Hi, just a reminder after three weeks as you asked! If it is possible, I am still interested in experimenting with the pre-trained networks. Thanks.

**Comment by ESanchezLozano** [comment]
Hi,

I had the chance to look into the folders and I regret to inform you that due to some maintenance carried out in the server where I had my GAN training most of the code and models were wiped out. I will try to keep looking for a backup but at this moment I reckon this might be lost. If I have the chance to re-train the models I will let you know. Note that this was not part of the paper but rather an extension of the code I freely developed on my own. I believe this version works better than that of the original paper so it would be good to have it re-trained again. If this is the case I will upload a model.  

I am sorry for that, and I apologise for the inconveniences. Still I really would encourage you to train the models as in the code using 300VW. You can additionally add some extra datasets with static images paired with perturbations of themselves. 

Please again accept my apologies for the inconveniences this might cause, I will post again if I get some better news regarding this topic.

**Comment by sanjeelparekh** [comment]
Thanks for looking into this again. 
I'll train with 300VW as you suggest. At the moment, I am trying to augment my previous dataset with another one.

**Comment by ESanchezLozano** [comment]
Hi,

I will try to add some extra bit of code to the training to account for the use of static images that may help you use any set of static datasets you might want to use.
